app/assets/javascripts/temp_pax_options.js

```diff
@@ -0,0 +1,22 @@
+$(document).on('turbolinks:load', function(){
+  $('#passenger_permanent').change(function(){
+    var registeredbyField = $('#pax_registered_with_DS');
+    var hasbrochureField = $('#pax_has_brochure');
+    if($(this).is(':checked')){
+      registeredbyField.prop('disabled', true);
+      registeredbyField.prop('checked', false);
+    }
+    else {
+      registeredbyField.prop('disabled', false);
+      registeredbyField.prop('checked', false);
+    }
```

Comment:
You could boil down this conditional down to:

```js
registeredbyField.prop('disabled', $(this).is(':checked'));
registeredbyField.prop('checked', false);
```

and the same for the other field.

Reply:
Without the if/else? I've been trying to figure out how to re-write these lines but I'm doing something wrong ...

Reply:
Correct. You can completely replace the conditional with those two lines.

`$(this).is(':checked')` | disabled | checked
--- | --- | ---
true | true | false
false | false | false

So you can always set `'checked'` to false and you always set `'disabled'` to match `$(this).is(':checked')`.